Documentation on custom activity (#310)
Unity-Technologies/com.unity.mobile.notifications: com.unity.mobile.notifications/Documentation~/Settings.md

### Custom Activity

> You can enable the **Use Custom Activity** option to override the activity that opens when the user taps the notification. By default, your app will use the only active activity in your app, if there is more than one activity, then it will use either `UnityPlayerActivity` or `UnityPlayerGameActivity` (since Unity 2023.1), depending on which one is available.

Reviewer: ```suggestion
You can enable the **Use Custom Activity** option to override the activity that opens when the user taps the notification. By default, your app will only use the active activity in your app. If more than one activity is available, it will either use `UnityPlayerActivity` or `UnityPlayerGameActivity` (Starting from Unity 2023.1), depending on their availability.
```

Author: I'm not sure this edit is clear. The logic is:
- if there is only one active activity in the game, we use it
- if there is more than one activity, we use UnityPlayerActivity or UnityPlayerGameActivity, whichever one is present
- we fail if there is more than one activity and either both UnityPlayerActivity or UnityPlayerGameActivity are present, or neither of them.

Reviewer: ```suggestion
You can enable the **Use Custom Activity** option to override the activity that opens when the user taps the notification. By default, your app will only use one active activity. If there is more than one activity, the app will use either UnityPlayerActivity or UnityPlayerGameActivity whichever is active. If neither or both UnityPlayerActivity or UnityPlayerGameActivity are active, Unity cannot determine which activity your app can use.
```
